compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEControlCodeToNpu.cpp

```diff
@@ -187,8 +109,16 @@ struct HalfDmaCpyNdToNpuConverter final
     staticStrides.insert(staticStrides.begin(),
                          numIntraAddrDim - staticStrides.size(), 0);
 
-    bool useNextBd{false};
+    bool useNextBd = op.getUseNextBd();
     int32_t nextBd{0};
+    if (useNextBd) {
+      std::optional<AMDAIE::BdIdOp> nextBdIdOp = op.getNextBdIdOp();
+      if (!nextBdIdOp) {
+        return op.emitOpError() << "useNextBd set, but no next BD ID op found";
+      }
```

Comment:
```suggestion
      if (!nextBdIdOp)
        return op.emitOpError() << "useNextBd set, but no next BD ID op found";
```
Btw, we follow this style guide: https://google.github.io/styleguide/cppguide.html